conf/base.config

```diff
@@ -11,7 +11,6 @@
 
 process {
 
-  // TODO nf-core: Check the defaults for all processes
   cpus = { check_max( 1 * task.attempt, 'cpus' ) }
   memory = { check_max( 7.GB * task.attempt, 'memory' ) }
```

Comment:
Maybe in this or another PR we should raise the max memory and max run times

to 16GB and 24h at least for the processes comet_db_search and predict_neoeptiopes_from_vcf

Reply:
Issue: #94

Reply:
https://github.com/Zethson/MHCquant/commit/8196505bb518d9e03bed0620df40783e69850d10 and https://github.com/Zethson/MHCquant/commit/154197eb06732c3f9b8c708cea32bc5ef2abfe81